libbeat/scripts/Makefile

```diff
@@ -51,7 +51,7 @@ build: $(GOFILES)
 # Create test coverage binary
 .PHONY: buildbeat.test
 buildbeat.test: $(GOFILES)
-	go test -c -covermode=atomic -coverpkg ${GOPACKAGES_COMMA_SEP}
+	go test -c -race -covermode=atomic -coverpkg ${GOPACKAGES_COMMA_SEP}
```

Comment:
I didn't even attend it to add it as part of this PR. I suggest I make it a variable which is off by default, but for example enabled for our daily builds.